src/DIRAC/DataManagementSystem/Client/DataManager.py

```diff
@@ -53,30 +47,28 @@ def _isOlderThan(stringTime, days):
     return False
 
 
-def _initialiseAccountingObject(operation, se, files):
-    """create accouting record"""
-    accountingDict = {}
-    accountingDict["OperationType"] = operation
+def _initialisebaseDict(operation, se, files):
```

Comment:
```suggestion
def _initialiseBaseMonitoringDict(operation, se, files):
```
(be as explicit as possible)

Reply:
That function is used for also Accounting, though. The dict created through it could also be sent to Accounting by DataOperationSender